src/screens/Exchange/Exchange.js

```diff
@@ -716,13 +778,36 @@ class ExchangeScreen extends React.Component<Props, State> {
   generateAssetsOptions = (assets) => {
     const { balances, paymentNetworkBalances } = this.props;
     const assetsList = Object.keys(assets).map((key: string) => assets[key]);
-    const nonEmptyAssets = assetsList.filter((asset: any) => {
-      return getBalance(balances, asset.symbol) !== 0 || asset.symbol === ETH;
+    const fiatCurrencies = [
```

Comment:
Maybe it's better to extract this to some config file?

Reply:
0291e7e